### PR TITLE
zookeeper and hdfs: alarms and dashboard_info

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

This PR adds:
- [X] `zookeeper` alarms
- [X] `hdfs` alarms
- [X] `zookeeper` module description/icon
- [X] `hdfs` module description/icon

`zookeeper` alarms:
 - availability

`hdfs` name node alarms:
 - availability
 - capacity usage
 - missing blocks
 - stale nodes
 - dead nodes

`hdfs` data node alarms:
 - availability
 - capacity usage
 - failed volumes

##### Component Name
[/web/gui](https://github.com/netdata/netdata/tree/master/web/gui)
[health/health.d](https://github.com/netdata/netdata/tree/master/health/health.d)

##### Additional Information

 - i tested the PR installing it, it works - i see all info/alarms on the dashboard.

cc: @killerwhile

### CODE REVIEW

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -504,6 +504,18 @@ netdataDashboard.menu = {
...
+    'zookeeper': {
+        title: 'Zookeeper',
+        icon: '<i class="fas fa-database"></i>',
+        info: 'Provides health statistics for <b><a href="https://zookeeper.apache.org/">Zookeeper</a></b> server. Data collected through the command port using <code><a href="https://zookeeper.apache.org/doc/r3.4.13/zookeeperAdmin.html#sc_zkCommands">mntr</a></code> command.'
+    },
+
+    'hdfs': {
+        title: 'HDFS',
+        icon: '<i class="fas fa-folder-open"></i>',
+        info: 'Provides <b><a href="https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html">Hadoop Distributed File System</a></b> performance statistics. Module collects metrics over <code>Java Management Extensions</code> through the web interface of an <code>HDFS</code> daemon.'
```
Comment (thiagoftsm) [inline web/gui/dashboard_info.js]: I have a doubt related to this link, I saw that for zookeper you are using firstly the `index` ( https://zookeeper.apache.org/ ) and in front you use a link to the version 3.4.13(https://zookeeper.apache.org/doc/r3.4.13/zookeeperAdmin.html#sc_zkCommands ), on the other hand, when you are setting links to Hadoop, you set for an old version. Why are you not using here a new version like the 3.2 ( http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html )? I have the fear we are missing important information from the newest version.

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -2625,5 +2637,17 @@ netdataDashboard.context = {
...
+    // ------------------------------------------------------------------------
+    // Zookeeper
+
+    'zookeeper.server_state': {
+        info:
+            '<code>0</code>: unknown; ' +
+            '<code>1</code>: leader; ' +
+            '<code>2</code>: follower; ' +
+            '<code>3</code>: observer; ' +
+            '<code>4</code>: standalone.'
```
Comment (thiagoftsm) [inline web/gui/dashboard_info.js]: From 0 to 3 you finished the line using semi-colon (`;`) , but for the last code you are finishing the line using  dot(`.`), Jasek already showed me that JS has been more flexible nowadays, but I confess I never heard nothing about to finish a command line with dots, is this acceptable nowadays?

Comment (ilyam8) [reply web/gui/dashboard_info.js]: it is just a text

File: web/gui/dashboard_info.js (hunk 1 above)
Comment (ilyam8) [reply web/gui/dashboard_info.js]: thx, changed links

File: web/gui/dashboard_info.js (hunk 2 above)
Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: All right, thank you!

**Comment by ilyam8** [comment]
@cakrit this is ready

**Pull request merged**